common/lib/capa/capa/inputtypes.py

```diff
@@ -629,8 +629,11 @@ def get_attributes(cls):
                                              #   set state
             Attribute('width', "400"),       # iframe width
             Attribute('height', "300"),      # iframe height
-            Attribute('sop', None)           # SOP will be relaxed only if this
-                                             # attribute is set to false.
+            # Title for the iframe, should be supplied by the author of the problem. Not translated
+            # because default display name is also not translated, and we can't import Django here anyway.
+            Attribute('title', "Custom Javascript Display Iframe"),
```

Comment:
I thought we should differentiate the iFrame in some way from its container. Having an iframe with the same title as the overall problem doesn't seem very useful.

Note that there will likely be text within the problem, outside of the iframe (as in the current example). With cone & cube, the iframe only hosts the block with the spinning objects.